std_msgs/String task
std_msgs/String req
---
zeabus_vision/vision_roulette data
